test/e2e/network_connect_disconnect_test.go

```diff
@@ -162,7 +166,11 @@ var _ = Describe("Podman network connect and disconnect", func() {
 
 		con2 := podmanTest.Podman([]string{"network", "connect", netName, "test"})
 		con2.WaitWithDefaultTimeout()
-		Expect(con2).Should(ExitWithError())
+		if os.Getenv("PODMAN_DB") == "boltdb" {
```

Comment:
this will not pass locally as the env is not set, you should use `podmanTest.DatabaseBackend == "..."`

Reply:
Nice catch, thank you. Fixed, repushed (with only that one-line change), and CI is green.